packages/ui/src/components/va-data-table/hooks/useStyleable.ts

```diff
@@ -2,7 +2,24 @@ import { TableCell } from './useRows'
 import { getFocusColor, getHoverColor } from '../../../services/color-config/color-functions'
 import { getColor } from '../../../services/color-config/color-config'
 import { computed, Ref } from 'vue'
-import { TableColumn } from './useColumns'
+import { TableColumn, TClassesOptions, TStyleOptions } from './useColumns'
+
+function getClasses (classes: TClassesOptions | undefined): string[] {
+  if (!classes) { return [] }
```

Comment:
I think it's worth using default value instead of checking for undefined
`function getClasses (classes: TClassesOptions | undefined = []): string[]`

Reply:
Thanks! Fixed it.